Source/WTF/wtf/PlatformHave.h

```diff
@@ -1371,6 +1371,11 @@
 #define HAVE_VPIO_DUCKING_LEVEL_API 1
 #endif
 
+// Only jsc-armv7 and jsc-mips bots do not support std::span.
+#if !(((CPU(ARM_THUMB2) && CPU(ARM_HARDFP)) || CPU(MIPS)) && OS(LINUX))
+#define HAVE_STD_SPAN 1
+#endif
```

Comment:
This should probably not be necessary.

Not sure why jsc-armv7 and jsc-mips bots are not building with C++20.

Reply:
I have a better solution by testing the GNU compiler via CMake to see if it has `<span>`.